dijet/config/cutflow_variables.py

```diff
@@ -35,10 +33,10 @@ def add_cutflow_variables(config: od.Config) -> None:
         "pt_avg": (200, 0, 1000),
         "alpha": (100, 0, 1),
     }
-    var_unit = {
-        "pt": "GeV",
-        "pt_avg": "GeV",
-    }
+    # var_unit = {
+    #     "pt": "GeV",
+    #     "pt_avg": "GeV",
+    # }
```

Comment:
In principle the unit should be set for `pt` and `pt_avg`, so you could actually comment this in and use it further down.